connector/src/main/java/org.apache.flink/connector/nebula/connection/NebulaStorageConnectionProvider.java

```diff
@@ -63,4 +63,37 @@ public StorageClient getStorageClient() throws Exception {
         return storageClient;
     }
 
+    public MetaClient getMetaClient() throws Exception {
```

Comment:
maybe you can create another java file such as Nebula Meta Connection Provider to support this method